docs/content/en/main/concepts/syncer.md

```diff
@@ -164,36 +195,41 @@ You can run the syncer in a kind cluster for development.
    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/kubernetes-sigs/kind/main/site/static/examples/kind-with-registry.sh)"
    ```
 
-1. Install `ko`:
-
-    ```sh
-    go install github.com/google/ko@latest
+1. From the kcp root directory:
+    ```bash
+    export KUBECONFIG=.kcp/admin.kubeconfig
     ```
 
-1. Build image and push to the local registry integrated with `kind`:
+1. Create a location workspace and immediately enter it:
 
     ```sh
-    KO_DOCKER_REPO=localhost:5001 ko publish ./cmd/syncer -t <image tag>
+    $ kubectl kcp workspace create location --enter
+    Workspace "location" (type root:organization) created. Waiting for it to be ready...
+    Workspace "location" (type root:organization) is ready to use.
+    Current workspace is "root:location" (type "root:organization").
     ```
 
-    By default `ko` will build for `amd64`. To build for `arm64`
-    (e.g. apple silicon), specify `--platform=linux/arm64`.
+1. To create the synctarget and use the image pushed to the local registry, supply `<image name>` to the
+   `kcp workload sync` plugin command, where `<image name>` was captured in the build steps, above.
 
+    ```sh
+    kubectl kcp workload sync kind --syncer-image <image name> -o syncer.yaml
+    ```
+    
 1. Create an organisation and immediately enter it:
 
     ```sh
+    $ kubectl kcp workspace ..
     $ kubectl kcp workspace create my-org --enter
```

Comment:
```suggestion
    $ kubectl kcp workspace create my-workloads --enter
```

Reply:
I have applied this change. The only issue is that "my-org" is used elsewhere on this page, and it is termed an organisation (as opposed to a second workspace as you have advised) in those sections. I decided to leave "my-org" in the steps in those other sections because a) those sections do not (yet) use the 2 workspace (locations/workloads) setup, so it might be confusing to see "my-locations" without reference to a "my-workloads" and b) I didn't really want to change those steps without testing them and ensuring I wasn't breaking something.

Anyway, let me know if you are happy to go with the suggested changes or whether you think it would be better to wait for the references to "my-org" to be removed and the 2 workspace format put in in all sections in the same PR. (My preference is just to go ahead and come back to it again later.)

Reply:
> My preference is just to go ahead and come back to it again later

I agree with that as well.